target_chains/ethereum/entropy_sdk/solidity/MockEntropy.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity ^0.8.0;
+
+interface IEntropyConsumer {
+    function _entropyCallback(
+        uint64 sequenceNumber,
+        address provider,
+        bytes32 randomNumber
+    ) external;
+}
+
+contract MockEntropy {
+    uint64 public sequenceNumber;
+    uint128 public constant FEE = 0.000015 ether;
```

Comment:
please make this a variable with setter and getter. a dApp protocol may want to test whether the changes in fee are automatically reflected in their requests or not.
Also you can add this in the constructor.

Reply:
Added setter and defined in constructor